it/mail/chrome/messenger/custom.properties

```diff
@@ -1,2 +1,2 @@
-headerRemoved=You have removed header from the list. This may require resetting of the search filter criteria.
-colonInHeaderName=Uno dei filtri utilizza un intestazione personalizzata contenente caratteri non validi, quali ad esempio ':', caratteri non stampabili, caratteri non ASCII, caratteri nel campo di valori 128-255. Rimuovere i caratteri non validi e riprovare.
+headerRemoved=Hai rimosso l'intestazione dalla lista. Ciò potrebbe richiedere il ripristino dei criteri del filtro di ricerca.
+colonInHeaderName=L'intestazione che hai inserito contiene un carattere non valido, come ':', carattere non stampabile, carattere non ASCII o un carattere ASCII a 8 bit. Rimuovi il carattere non valido e riprova.
```

Comment:
CORRECTIONS NEEDED. 

I would rephrase this sentence as follows:  +colonInHeaderName=L'intestazione che hai inserito contiene un carattere non valido (ad esempio ':'), un carattere non stampabile, un carattere non ASCII o un carattere ASCII a 8 bit. Rimuovi il carattere non valido e riprova.